pipe.go

```diff
@@ -181,9 +181,12 @@ func DialPipe(path string, timeout *time.Duration) (net.Conn, error) {
 		return nil, err
 	}
 
-	if state&cPIPE_READMODE_MESSAGE != 0 {
+	/**
+	Windows support message type pipes in message-read mode only. Removing this check to allow for windows named pipes.
```

Comment:
Looks like this message could be better put as a commit message, and the code removed?